drivers/sx126x/sx126x.c

```diff
@@ -207,6 +207,9 @@ int sx126x_init(sx126x_t *dev)
         DEBUG("[sx126x] init: cmd status %d\n", radio_status.cmd_status);
     }
 
+    /* Radio Rx timeout timer stopped on preamble detection */
```

Comment:
this commit should be part of https://github.com/RIOT-OS/RIOT/pull/16599. Please cherry pick commit there and rebase again.

Reply:
Done.